db-facilities/facdb/sql/dsny_specialwastedrop.sql

```diff
@@ -4,8 +4,8 @@ SELECT uid,
     name as facname,
     parsed_hnum as addressnum,
     parsed_sname as streetname,
-    address,
-    city,
+    NULL as address,
+    NULL as city,
```

Comment:
These seem to still be there, looking at the files in recipes

Reply:
You're right, definitely shouldn't have dropped these. Will remove this change and re-run